File: PHP/mahasiswa.php
<?php
class mahasiswa 
{
    private $nim = "";
    private $nama = "";
    private $prodi = "";
    private $fakultas = "";
    function __construct($n, $na, $p, $f)
    {
        $this->nim = $n;
        $this->nama = $na;
        $this->prodi = $p;
        $this->fakultas = $f;
    }
    function setNim($a)
    {
        $this->nim = $a;
    }
    function setNama($a)
    {
        $this->nama = $a;
    }
    function setProdi($a)
    {
        $this->prodi = $a;
    }
    function setFakultas($a)
    {
        $this->fakultas = $a;
    }

    function getNim()
    {
        return $this->nim;
    }
    function getNama()
    {
        return $this->nama;
    }
    function getProdi()
    {
        return $this->prodi;
    }
    function getFakultas()
    {
        return $this->fakultas;
    }
    function __destruct(){
        
    }
}
?>

File: Python/Mahasiswa.py
class Mahasiswa: 
    __nim = ""
    __nama  = ""
    __prodi_= ""
    __fakultas = ""

    def __init__(self):
        self.nim = ""
        self.nama = ""
        self.prodi = ""
        self.fakultas = ""

    def setNim(self, a):
        self.nim = a
    def setNama(self, a):
        self.nama = a
    def setProdi(self, a):
        self.prodi = a
    def setFakultas(self, a):
        self.fakultas = a
    
    def getNim(self):
        return self.nim
    def getNama(self):
        return self.nama
    def getProdi(self):
        return self.prodi
    def getFakultas(self):
        return self.fakultas

    
    

